CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### AppCenterAnalytics
 
 * **[Feature]** Preparation work for a future change in transmission protocol and endpoint for Analytics data. There is no impact on your current workflow when using App Center.
+* **[Fix]** Fix an bug where nested custom properties for an event would not pass validation.
```

Comment:
Do we include 1ds fixes in the GitHub change log?

Reply:
If you ask me, yes. We're not 100% consistent there but I included it to make writing release notes for the next release easier.